packages/dds/tree/src/test/shared-tree/treeNodeApi.spec.ts

```diff
@@ -47,7 +47,7 @@ describe("treeApi", () => {
 		/**
 		 * Runs a set of transaction tests, either passing the TreeView or the root node to the `runTransaction` function depending on the configuration.
 		 * @remarks This allows for code coverage of both of those variants of the `runTransaction` API without duplicating these tests entirely.
-		 * */
+		 */
```

Comment:
I was bulk fixing issues like this in comments from the tag removal. I so no reason to revert this unrelated cleanup.